apps/web/src/actions/datasets/create.ts

```diff
@@ -0,0 +1,46 @@
+'use server'
+
+import { createDataset } from '@latitude-data/core/services/datasets/create'
+import disk from '$/lib/disk'
+import { z } from 'zod'
+
+import { withProject } from '../procedures'
+
+const ACCEPTED_FILE_TYPES = [
+  'text/csv',
+  'application/vnd.ms-excel',
+  'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+  'application/vnd.oasis.opendocument.spreadsheet',
+]
```

Comment:
I have to investigate but I'm sure there is a nodejs package to read any of these formats. So we can allow anyone

Reply:
Looks this one is the most popular https://github.com/adaltas/node-csv looks nice